build.sbt

```
@@ -15,20 +15,25 @@ ThisBuild / tlCiReleaseBranches := Seq("main")
 // true by default, set to false to publish to s01.oss.sonatype.org
 ThisBuild / tlSonatypeUseLegacyHost := true
 
-ThisBuild / crossScalaVersions := Seq("2.12.18", "2.13.11", "3.3.0")
-ThisBuild / scalaVersion := "3.3.0"
+val scala213 = "2.13.12"
+val scala3 = "3.3.1"
+ThisBuild / crossScalaVersions := Seq(scala213, scala3)
+ThisBuild / scalaVersion := scala3
```

since we now have full control over this, I opine that the default build be 2.13. but that can be part of a "rebrand" PR, so to speak

```suggestion
ThisBuild / scalaVersion := scala213
```

---

This might require a githubWorkflowRegenerate when I merge it, but here goes.